tests/bson/bson-utcdatetime_error-001.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+BSON BSON\UTCDateTime #001 error
+--INI--
+date.timezone=America/Los_Angeles
+--SKIPIF--
+<?php if (defined("HHVM_VERSION_ID")) exit("skip HHVM handles parameter parsing differently"); ?>
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; CLEANUP(STANDALONE) ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(STANDALONE);
+
+$classname = BSON_NAMESPACE . "\\UTCDateTime";
+
+throws(function() use($classname) {
+    $d = new $classname;
```

Comment:
$classname is defined outside of the closure for no apparent reason, which then requires use($classname).

Defining $classname inside the closure seems like would be simpler.

It might have made sense in the original test cases if the variable was used in couple of places, but in this test where it is only used once from within the closure it looks funny to declare it outside and then use() it.

Did that clarify the comment?